src/commands/schemaUpload.ts

```diff
@@ -145,11 +143,7 @@ export async function uploadNewSchema(item: vscode.Uri) {
 
     // get the new schema to pop in the view by getting the treeitem to reveal
     // the schema's item.
-    schemaViewProvider.revealSchema(results[1] as Schema);
-  } else {
```

Comment:
No longer need to do (too late to be always noticed by the user) in the else block. Will have already been done within `updateRegistryCacheAndFindNewSchema`.